src/strategies/email-password.ts

```
@@ -3,31 +1,37 @@
+    email: formData.email,
+    password: formData.password,
+    options: {
+      data: {
+        firstName: formData.firstName,
+        lastName: formData.lastname,
+      },
+    },
+  });
+
+  // ~ ======= create user if account was created successfully -->
+  if (data.user && data.user.email) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)